app/Services/GameService.php

```diff
@@ -21,12 +21,11 @@ function getGamesById (int $id)
     // todo: add optional params for orderBy, direction, and limit
     function getGames()
     {
-        return
-            [
-                Game::make(['id' => 1, 'title' => 'Monopoly', 'rating' => 1]),
-                Game::make(['id' => 2, 'title' => 'Risk', 'rating' => 2]),
-                Game::make([ 'id' => 3, 'title' => 'DND', 'rating' => 5]),
-            ];
+        $allGames = Game::all('id', 'title', 'rating', 'age', );
```

Comment:
your error is from selecting a column that doesn't exist. you don't need specify columns here, just use `Game::all()`